src/javascript/main.js

```diff
@@ -0,0 +1,60 @@
+var Minispot = window.Minispot || {};
+
+var response;
+
+Minispot.search = function() {
+
+	var search_button = document.querySelector('#search');
+
+	search_button.addEventListener('click', function() {
+
+		var query = document.querySelector('#query').value;
+		var data = ""
+		ajax(query, data, displayResult);
+		
+	});
+}
+
+function ajax(query, data, callback) {
+
+	var req = new XMLHttpRequest();
+	var url = "https://api.spotify.com/v1/search/?type=track&q=" + query;
+
+	req.open("GET", url);  
+	req.send(data); 
+	req.onload = function(event) {
```

Comment:
In here you could check if the request was successful (`this.status === 200`) and display an error or something otherwise. 

You can also access the response via `this.response`.